specification/gedcom-3-structures-1-organization.md

```diff
@@ -1229,6 +1229,7 @@ n REPO @<XREF:REPO>@                       {1:1}  g7:REPO
   +1 CALN <Special>                        {0:M}  g7:CALN
      +2 MEDI <Enum>                        {0:1}  g7:MEDI
         +3 PHRASE <Text>                   {0:1}  g7:PHRASE
+  +1 DATE <DateValue>                      {0:1}  g7:DATE
```

Comment:
```suggestion
  +1 DATE <DateValue>                      {0:M}  g7:DATE
```

Couldn't we have several dates as it was transferred to and from the same holder?

Reply:
Don't have a strong case for M currently, but changing to M in future would be non-breaking whereas changing from M to 1 would be a breaking change, so discussed staying with 1 for now.